src/v2/Apps/ArtistSeries/ArtistSeriesApp.tsx

```diff
@@ -24,6 +25,8 @@ const ArtistSeriesApp: React.FC<ArtistSeriesAppProps> = ({ artistSeries }) => {
     const { railArtist } = artistSeries
     return (
       <AppContainer maxWidth="100%">
+        {/* NOTE: react-head automatically moves these tags to the <head> element */}
```

Comment:
What I'll typically do here is a wrap everything in fragment and move the head tag stuff outside of the container, just so I know there's no UI being included and it kind of makes sense hierarchically — but very much a preference thing, I don't think we have any conventions for this. 👍